apps/music/index.html

```diff
@@ -63,28 +62,72 @@ <h1 id="title-text"><bdi data-l10n-id="music"></bdi></h1>
     </gaia-header>
     <div id="views">
       <div id="views-tiles" class="view bb-tabpanel hidden" aria-labelledby="tabs-mix" role="tabpanel">
-        <form id="views-tiles-search" role="search" class="skin-dark">
-          <button id="views-tiles-search-close" data-l10n-id="search-close" type="submit">close</button>
-          <p>
-            <input id="views-tiles-search-input" type="search" dir="auto"
-                   x-inputmode="verbatim"
-                   data-l10n-id="search-music" placeholder="SearcH MusiC">
-            <button id="views-tiles-search-clear" type="reset">Clear</button>
-          </p>
-        </form>
-        <div id="views-tiles-anchor"></div>
+        <div id="views-scrollable-tiles-anchor">
+          <form id="views-tiles-search" role="search" class="skin-dark">
+            <button id="views-tiles-search-close" data-l10n-id="search-close" type="submit">close</button>
+            <p>
+              <input id="views-tiles-search-input" type="search" dir="auto"
+                     x-inputmode="verbatim"
+                     data-l10n-id="search-music" placeholder="SearcH MusiC">
+              <button id="views-tiles-search-clear" type="reset">Clear</button>
+            </p>
+          </form>
+          <div id="views-tiles-anchor"></div>
+        </div>
+        <!-- BB tabs -->
+        <div class="tabs">
+          <ul role="tablist" class="bb-tablist skin-dark">
+            <li role="presentation">
+              <a data-option="mix" role="tab" aria-selected="false" aria-controls="views-tiles" data-icon="grid" data-l10n-id="mix-tab"></a>
+            </li>
+            <li role="presentation">
+              <a data-option="playlist" role="tab" aria-selected="false" aria-controls="views-list" data-icon="playlist" data-l10n-id="playlists-tab"></a>
+            </li>
+            <li role="presentation">
+              <a data-option="artist" role="tab" aria-selected="false" aria-controls="views-list" data-icon="artist" data-l10n-id="artists-tab"></a>
+            </li>
+            <li role="presentation">
+              <a data-option="album" role="tab" aria-selected="false" aria-controls="views-list" data-icon="album" data-l10n-id="albums-tab"></a>
+            </li>
+            <li role="presentation">
+              <a data-option="title" role="tab" aria-selected="false" aria-controls="views-list" data-icon="songs" data-l10n-id="songs-tab"></a>
+            </li>
+          </ul>
+        </div>
```

Comment:
Yes, It would be nice to turn the tabs into a web component, and I heard some folks are working on the gaia components repository for v3, maybe we can ask them to make the tabs component for gaia, if there is any other app is going to use in the future, or we can make it by our own which could be a separate bug.